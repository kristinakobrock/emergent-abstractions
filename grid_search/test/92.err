  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  4%|###7                                                                                         | 2/50 [00:00<00:02, 18.52it/s] 10%|#########3                                                                                   | 5/50 [00:00<00:02, 22.31it/s] 16%|##############8                                                                              | 8/50 [00:00<00:01, 25.33it/s] 22%|####################2                                                                       | 11/50 [00:00<00:01, 25.45it/s] 30%|###########################6                                                                | 15/50 [00:00<00:01, 26.87it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 23.35it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 22.74it/s] 48%|############################################1                                               | 24/50 [00:01<00:01, 23.43it/s] 54%|#################################################6                                          | 27/50 [00:01<00:00, 24.99it/s] 60%|#######################################################2                                    | 30/50 [00:01<00:00, 26.33it/s] 66%|############################################################7                               | 33/50 [00:01<00:00, 26.11it/s] 72%|##################################################################2                         | 36/50 [00:01<00:00, 25.93it/s] 78%|#######################################################################7                    | 39/50 [00:01<00:00, 24.48it/s] 84%|#############################################################################2              | 42/50 [00:01<00:00, 23.87it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 26.88it/s] 98%|##########################################################################################1 | 49/50 [00:01<00:00, 25.39it/s]100%|############################################################################################| 50/50 [00:02<00:00, 24.76it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 25.20it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 24.68it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 26.74it/s]100%|############################################################################################| 13/13 [00:00<00:00, 31.38it/s]100%|############################################################################################| 13/13 [00:00<00:00, 29.19it/s]
