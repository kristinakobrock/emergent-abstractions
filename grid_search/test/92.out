# ["--attributes=5", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6929397583007812, "acc": 0.594485878944397, "length": 3.504488229751587, "mode": "train", "epoch": 1}
{"loss": 0.6926109790802002, "acc": 0.5105469226837158, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6686409115791321, "acc": 0.6098987460136414, "length": 3.3135197162628174, "mode": "train", "epoch": 2}
{"loss": 0.6808556318283081, "acc": 0.5947265625, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6639863848686218, "acc": 0.6269479990005493, "length": 3.3512957096099854, "mode": "train", "epoch": 3}
{"loss": 0.6796345710754395, "acc": 0.6058593392372131, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6615837812423706, "acc": 0.6238110065460205, "length": 3.322864055633545, "mode": "train", "epoch": 4}
{"loss": 0.6766974925994873, "acc": 0.6187500357627869, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6602073311805725, "acc": 0.6269933581352234, "length": 3.33805513381958, "mode": "train", "epoch": 5}
{"loss": 0.6727985143661499, "acc": 0.6228516101837158, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6586022973060608, "acc": 0.6269800066947937, "length": 3.356393575668335, "mode": "train", "epoch": 6}
{"loss": 0.6505464315414429, "acc": 0.6399002075195312, "length": 3.01953125, "mode": "test", "epoch": 6}
{"loss": 0.6566591262817383, "acc": 0.6269855499267578, "length": 3.4108777046203613, "mode": "train", "epoch": 7}
{"loss": 0.6421979665756226, "acc": 0.6458984017372131, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.6508057713508606, "acc": 0.6308659911155701, "length": 3.552428722381592, "mode": "train", "epoch": 8}
{"loss": 0.6357787847518921, "acc": 0.6517577767372131, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.6404326558113098, "acc": 0.639567494392395, "length": 3.7219796180725098, "mode": "train", "epoch": 9}
{"loss": 0.6264117956161499, "acc": 0.6517577767372131, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.6270691156387329, "acc": 0.6539875268936157, "length": 3.855362892150879, "mode": "train", "epoch": 10}
{"loss": 0.6084109544754028, "acc": 0.6771484017372131, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.6153095960617065, "acc": 0.6630920171737671, "length": 3.9080419540405273, "mode": "train", "epoch": 11}
{"loss": 0.6048849821090698, "acc": 0.6769530773162842, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.6055223345756531, "acc": 0.6734438538551331, "length": 3.914321184158325, "mode": "train", "epoch": 12}
{"loss": 0.604646623134613, "acc": 0.6832031607627869, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.5964010953903198, "acc": 0.6781122088432312, "length": 3.942031145095825, "mode": "train", "epoch": 13}
{"loss": 0.5995169878005981, "acc": 0.6902344226837158, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.5815595984458923, "acc": 0.6913461089134216, "length": 3.957158088684082, "mode": "train", "epoch": 14}
{"loss": 0.5946906208992004, "acc": 0.6912109851837158, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.5676052570343018, "acc": 0.7015202641487122, "length": 3.9640421867370605, "mode": "train", "epoch": 15}
{"loss": 0.5961885452270508, "acc": 0.6896483898162842, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.5545245409011841, "acc": 0.7117801904678345, "length": 3.9732015132904053, "mode": "train", "epoch": 16}
{"loss": 0.5934739112854004, "acc": 0.6910156607627869, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.544408917427063, "acc": 0.715200662612915, "length": 3.975701332092285, "mode": "train", "epoch": 17}
{"loss": 0.5747900009155273, "acc": 0.7037109732627869, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.5330057740211487, "acc": 0.7199432253837585, "length": 3.9829893112182617, "mode": "train", "epoch": 18}
{"loss": 0.5663977861404419, "acc": 0.7052733898162842, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.5258663892745972, "acc": 0.728200376033783, "length": 3.988010883331299, "mode": "train", "epoch": 19}
{"loss": 0.5647661089897156, "acc": 0.6964843273162842, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.5202879309654236, "acc": 0.7309195399284363, "length": 3.9899373054504395, "mode": "train", "epoch": 20}
{"loss": 0.5502947568893433, "acc": 0.7144531607627869, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.5103272199630737, "acc": 0.7378124594688416, "length": 3.9909512996673584, "mode": "train", "epoch": 21}
{"loss": 0.5282232165336609, "acc": 0.7326171398162842, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.5023680925369263, "acc": 0.741003692150116, "length": 3.9939053058624268, "mode": "train", "epoch": 22}
{"loss": 0.5154709815979004, "acc": 0.7408203482627869, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.49559885263442993, "acc": 0.7448825240135193, "length": 3.9963645935058594, "mode": "train", "epoch": 23}
{"loss": 0.5191940069198608, "acc": 0.7380859851837158, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.4870527684688568, "acc": 0.7497832179069519, "length": 3.996141195297241, "mode": "train", "epoch": 24}
{"loss": 0.5267482399940491, "acc": 0.7304688096046448, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.48475685715675354, "acc": 0.7543519735336304, "length": 3.994093894958496, "mode": "train", "epoch": 25}
{"loss": 0.5180171728134155, "acc": 0.7339844107627869, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.4814727008342743, "acc": 0.7541697025299072, "length": 3.9984755516052246, "mode": "train", "epoch": 26}
{"loss": 0.517655611038208, "acc": 0.7326172590255737, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.47874265909194946, "acc": 0.7572593688964844, "length": 3.9970967769622803, "mode": "train", "epoch": 27}
{"loss": 0.5168622732162476, "acc": 0.7406249642372131, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.47510671615600586, "acc": 0.7590813040733337, "length": 3.9983673095703125, "mode": "train", "epoch": 28}
{"loss": 0.5151432156562805, "acc": 0.7374999523162842, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.4741172790527344, "acc": 0.7572278380393982, "length": 3.9988415241241455, "mode": "train", "epoch": 29}
{"loss": 0.5188989639282227, "acc": 0.7402344346046448, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.4697921574115753, "acc": 0.7607699036598206, "length": 3.997593879699707, "mode": "train", "epoch": 30}
{"loss": 0.5088922381401062, "acc": 0.7435546517372131, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.4661843776702881, "acc": 0.7630570530891418, "length": 3.9994893074035645, "mode": "train", "epoch": 31}
{"loss": 0.5133680105209351, "acc": 0.7408203482627869, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.4656413197517395, "acc": 0.7636783719062805, "length": 3.9993560314178467, "mode": "train", "epoch": 32}
{"loss": 0.5153079628944397, "acc": 0.7396484017372131, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.46132513880729675, "acc": 0.7682616710662842, "length": 3.999545097351074, "mode": "train", "epoch": 33}
{"loss": 0.5173642039299011, "acc": 0.7392578125, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.46115589141845703, "acc": 0.7655500173568726, "length": 3.999427318572998, "mode": "train", "epoch": 34}
{"loss": 0.5156674981117249, "acc": 0.7419920563697815, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.4579194188117981, "acc": 0.7661600708961487, "length": 3.998640537261963, "mode": "train", "epoch": 35}
{"loss": 0.5226843357086182, "acc": 0.7337890267372131, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.45247596502304077, "acc": 0.7703422904014587, "length": 3.99857234954834, "mode": "train", "epoch": 36}
{"loss": 0.511497437953949, "acc": 0.7396484017372131, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.4510073661804199, "acc": 0.7724500298500061, "length": 3.999290704727173, "mode": "train", "epoch": 37}
{"loss": 0.5063364505767822, "acc": 0.7453126311302185, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.44722846150398254, "acc": 0.7775420546531677, "length": 3.9997751712799072, "mode": "train", "epoch": 38}
{"loss": 0.5067325234413147, "acc": 0.7453125715255737, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.4442148804664612, "acc": 0.781875491142273, "length": 3.998616933822632, "mode": "train", "epoch": 39}
{"loss": 0.505549430847168, "acc": 0.7433595061302185, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.4396961033344269, "acc": 0.7827461361885071, "length": 3.999844789505005, "mode": "train", "epoch": 40}
{"loss": 0.5094856023788452, "acc": 0.7437500357627869, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.4348302185535431, "acc": 0.7876655459403992, "length": 3.9998085498809814, "mode": "train", "epoch": 41}
{"loss": 0.5013250112533569, "acc": 0.7466797232627869, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.4318254590034485, "acc": 0.7903268337249756, "length": 3.999871015548706, "mode": "train", "epoch": 42}
{"loss": 0.4988587498664856, "acc": 0.7552734017372131, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.4259207248687744, "acc": 0.7981069087982178, "length": 3.9987878799438477, "mode": "train", "epoch": 43}
{"loss": 0.5003048181533813, "acc": 0.7548827528953552, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.4224998652935028, "acc": 0.8016948699951172, "length": 3.9998362064361572, "mode": "train", "epoch": 44}
{"loss": 0.48929041624069214, "acc": 0.7671875357627869, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.4160972237586975, "acc": 0.8039586544036865, "length": 3.999955892562866, "mode": "train", "epoch": 45}
{"loss": 0.49092280864715576, "acc": 0.7718749642372131, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.4089186489582062, "acc": 0.8122707605361938, "length": 3.9998481273651123, "mode": "train", "epoch": 46}
{"loss": 0.48112547397613525, "acc": 0.7746093273162842, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.4049113690853119, "acc": 0.8127177953720093, "length": 3.9994893074035645, "mode": "train", "epoch": 47}
{"loss": 0.46496546268463135, "acc": 0.7822265625, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.3967883884906769, "acc": 0.8189471364021301, "length": 3.999952554702759, "mode": "train", "epoch": 48}
{"loss": 0.47307154536247253, "acc": 0.7806640267372131, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.39112287759780884, "acc": 0.8257344365119934, "length": 3.9999446868896484, "mode": "train", "epoch": 49}
{"loss": 0.4660804271697998, "acc": 0.7888672351837158, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.3867686092853546, "acc": 0.8260102272033691, "length": 3.9996230602264404, "mode": "train", "epoch": 50}
{"loss": 0.46115338802337646, "acc": 0.7874999642372131, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.38341256976127625, "acc": 0.8262417316436768, "length": 3.9997007846832275, "mode": "train", "epoch": 51}
{"loss": 0.4656290113925934, "acc": 0.7876953482627869, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.3781938850879669, "acc": 0.8307480812072754, "length": 3.9992942810058594, "mode": "train", "epoch": 52}
{"loss": 0.4533507823944092, "acc": 0.7908202409744263, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.3714625835418701, "acc": 0.834498405456543, "length": 3.9997897148132324, "mode": "train", "epoch": 53}
{"loss": 0.4536021053791046, "acc": 0.7880859971046448, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.3701748847961426, "acc": 0.8369774222373962, "length": 3.9962007999420166, "mode": "train", "epoch": 54}
{"loss": 0.45818009972572327, "acc": 0.7941405773162842, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.364128440618515, "acc": 0.8386257290840149, "length": 3.9996581077575684, "mode": "train", "epoch": 55}
{"loss": 0.4553126394748688, "acc": 0.7939453125, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.3580206334590912, "acc": 0.8410958051681519, "length": 3.999124765396118, "mode": "train", "epoch": 56}
{"loss": 0.43723976612091064, "acc": 0.8076172471046448, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.3540603816509247, "acc": 0.8445644378662109, "length": 3.9997313022613525, "mode": "train", "epoch": 57}
{"loss": 0.43067896366119385, "acc": 0.8113280534744263, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.3472500145435333, "acc": 0.8477926850318909, "length": 3.9999375343322754, "mode": "train", "epoch": 58}
{"loss": 0.4370078444480896, "acc": 0.8144530653953552, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.3425842225551605, "acc": 0.8518592715263367, "length": 3.9997670650482178, "mode": "train", "epoch": 59}
{"loss": 0.4283639192581177, "acc": 0.8160156607627869, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.3383816182613373, "acc": 0.854622483253479, "length": 3.999354124069214, "mode": "train", "epoch": 60}
{"loss": 0.4283517301082611, "acc": 0.8226562142372131, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7234615087509155
